swifty_companion/Services/UserService.swift

```diff
@@ -33,7 +33,7 @@ final class UserService: IUserService {
         guard let url = URL(string: "https://api.intra.42.fr/v2/users?range[login]=\(searchString.lowercased()),\(searchString.lowercased())z") else { return }
         var urlRequest = URLRequest(url: url)
         let bearer = "Bearer \(accessToken)"
-//        print(accessToken)
+        print(accessToken)
```

Comment:
Usually you should not commit `print(..)` or any other debug code to "production" / repo